controllers/articles.controller.js

```diff
@@ -19,3 +19,24 @@ exports.getArticles = (req, res, next) => {
         next(err)
     })
 }
+
+exports.patchArticleVotesById = (req, res, next) => {
+    const { article_id } = req.params
+    const body = req.body
+
+    console.log(body.inc_votes)
+
+    if (!body.inc_votes) {
+        return res.status(400).send({ msg: 'Bad request: inc_votes is required' });
+      }
+
+      checkIfArticleExists(article_id)
+      .then(() => {
+        return updateArticleVotesById(body, article_id).then((article) => {
+            res.status(200).send({ article })
+        })
+      })
```

Comment:
You dont actually need to separately assert if the article exists and psql should throw an error if you tried an update with an article that doesnt exist